variables.tf

```diff
@@ -227,9 +220,22 @@ variable "private_endpoint_connection_name" {
 variable "subnet_sa" {
   description = "The subnet IDs"
   type        = list(string)
+  default     = []
 }
 
 variable "enable_data_lookup" {
   type    = bool
   default = false
 }
+
+variable "blob_resource_group_name" {
```

Comment:
in root module it says as string

Reply:
creating this variable in the locals, so removed the var definition from root module